lists2safebrowsing.py

```diff
@@ -454,37 +457,75 @@ def get_tracker_lists(config, section, chunknum):
+        return
+
+    if config.has_option(section, 'disconnect_url'):
+        edit_config(
+            config, section, option='disconnect_url',
+            old_value=initial_disconnect_url_val, new_value=version)
+
+    if config.has_option(section, 's3_key'):
+        versioned_key = 'tracking/{ver}/'.format(ver=version)
+        edit_config(
+            config, section, option='s3_key',
+            old_value=initial_s3_key_value, new_value=versioned_key)
+
+
+def revert_config(config, version):
+    edit_config(
+        config=config, section='main', option='default_disconnect_url',
+        old_value=version, new_value='master')
+    for section in config.sections():
+        versioning_needed = (
+            config.has_option(section, 'versioning_needed')
+            and config.get(section, 'versioning_needed')
+        )
+        if not versioning_needed:
+            continue
+        version_configurations(config, section, version, revert=True)
+
+
 def get_versioned_lists(config, chunknum, version):
-    '''
+    """
     Checks `versioning_needed` in each sections then versions the tracker lists
     by overwriting the existing SafeBrowsing formatted files.
-    '''
-    default_url = config.get('main', 'default_disconnect_url')
-    versioned_default_url = default_url.replace('master', version)
-    print('New default disconnect url is ' + versioned_default_url)
-    config.set('main', 'default_disconnect_url', versioned_default_url)
+    """
+    edit_config(
+        config, section='main', option='default_disconnect_url',
+        old_value='master', new_value=version)
     for section in config.sections():
         versioning_needed = (
             config.has_option(section, 'versioning_needed')
-                and config.get(section, 'versioning_needed')
+            and config.get(section, 'versioning_needed')
         )
         if not versioning_needed:
             continue
-        print('Versioning for ' + config.get(section, 'output'))
-        default = config.get('main', 'default_disconnect_url')
-        print('Default disconnect URL: ' + default)
-        if config.has_option(section, 'disconnect_url'):
-            disconnect_url = config.get(section, 'disconnect_url')
-            versioned_disconnect_url = disconnect_url.replace('master', version)
-            config.set(section, 'disconnect_url', versioned_disconnect_url)
-            new_url = config.get(section, 'disconnect_url')
-            print('Versioned disconnect url is ' + new_url)
-        if config.has_option(section, 's3_key'):
-            s3_key = config.get(section, 's3_key')
-            versioned_s3_key = s3_key.replace('tracking/', 'tracking/{ver}/'.format(ver=version))
-            config.set(section, 's3_key', versioned_s3_key)
-            new_s3_key = config.get(section, 's3_key')
-            print('Versioned s3 key is ' + new_s3_key)
+        print('Version {ver} for {output}'.format(
```

Comment:
nit: put an extra couple of white-space lines before this? The console output currently puts this line closer to the *previous* list rather than the one it's related to. E.g.,
```
Version 70.0 for base-track-digest256

------ tracking-protection-base ------
-->blocklist: https://raw.githubusercontent.com/mozilla-services/shavar-prod-lists/70.0/disconnect-blacklist.json)
 * filter ['Advertising', 'Analytics', 'Social', 'Disconnect'] matched 2016 domains
 * removing 1 rule(s) due to DNT exceptions
Tracking protection(tracking-protection-base): publishing 2016 items; file size 64534
Version 70.0 for content-track-digest256

------ tracking-protection-content ------
-->blocklist: https://raw.githubusercontent.com/mozilla-services/shavar-prod-lists/70.0/disconnect-blacklist.json)
 * filter ['Content'] matched 514 domains
 * removing 1 rule(s) due to DNT exceptions
Tracking protection(tracking-protection-content): publishing 513 items; file size 16438
```